variables.tf

```diff
@@ -51,8 +51,8 @@ variable "routes" {
   default     = []
 }
 
-variable "delete_default_internet_gateway_routes" {
-  description = "If set, ensure that all routes within the network specified whose names begin with 'default-route' and with a next hop of 'default-internet-gateway' are deleted"
+variable "delete_default_routes_on_create" {
```

Comment:
nit: I think we should add bool type and set as `false` since other variables seem to have it.  However this will be a breaking change, so I will defer to @morgante

Reply:
Yes we should switch to boolean type. It's not a breaking change since strings can be cast to booleans.

Reply:
Nice, I assumed even with typecasting it would be technically considered a breaking change. TIL